CompanyManagerProject/MainWindow.xaml.cs

```diff
@@ -23,6 +25,26 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
+            var connectionString = ConfigurationManager.ConnectionStrings["CompanyManagerProject.Properties.Settings.CompanyManagerConnectionString"].ConnectionString;
```

Comment:
To niepotrzebne, tworzenie połączenia z bazą danych odbywa się w warstwie DAL w kontekstach.